package.json

```diff
@@ -19,7 +20,8 @@
   "jest": {
     "setupTestFrameworkScriptFile": "<rootDir>/test/setupTests.js",
     "transformIgnorePatterns": [
-      "/node_modules/(?!test-component).+\\.js$"
+      "/node_modules/(?!test-component).+\\.js$",
+      "/node_modules/(?!MODULE_NAME_HERE).+\\.js$"
```

Comment:
what does this line do?

Reply:
It help jest for transforming any es2015 module used in developing to es5.

Reply:
This doesn't look right to me, `MODULE_NAME_HERE` seems like a placeholder for something else. i.e. you're supposed to put your module name in to solve an issue:

https://github.com/facebook/jest/issues/6229#issuecomment-403539460

Let's revert this change, as I don't think it's serving any functional purpose (we could also consider taking this line out entirely, perhaps, but as part of another pr)